### ISSUE TITLE
KFS on OSX failing after reboot

### ISSUE THREAD

**Issue by echtniels** [keybase/client#4176]
Keybase GUI Version: 1.0.17-20160829110024+c4e8dc8

When starting keybase through OSX gui, "Keybase Error" pops up:
"Task timed out: /Applications/Keybase.app/Contents/SharedSupport/bin/keybase -d --log-format=file install --format=json --components=service"

After this, client can login but KFS not working:

```
$ cd /keybase/public/chris
-bash: cd: /keybase/public/chris: Device not configured
$ open /keybase/public/chris
The file /keybase/public/chris does not exist.
```

---- 'more details' output -------
```
Installer: 1.1.36
Privileged Helper: Installed, Bundle Version: 1.0.29, Version: 1.0.29
Updater: Installed, PID: 6234
Service: Unknown, Install Error: Task timed out: /Applications/Keybase.app/Contents/SharedSupport/bin/keybase -d --log-format=file install --format=json --components=service
Fuse: Unknown, 
KBFS: Unknown, 
CLI: Unknown, 

Log:
Installables: (
    "<KBHelperTool: 0x7fae61636bc0>",
    "<KBUpdaterService: 0x7fae61444e80>",
    "<KBService: 0x7fae6144ca10>",
    "<KBFuseComponent: 0x7fae6144cc80>",
    "<KBFSService: 0x7fae6144da90>",
    "<KBCommandLine: 0x7fae6144dcc0>"
)
Install: Privileged Helper
Install: Updater
Task: /Applications/Keybase.app/Contents/SharedSupport/bin/keybase -d --log-format=file install --format=json --components=updater
Task termination handler, reading output
Task /Applications/Keybase.app/Contents/SharedSupport/bin/keybase -d --log-format=file install --format=json --components=updater (out): {
  "componentResults": [
    {
      "name": "updater",
      "status": {
        "code": 0,
        "name": "OK",
        "desc": "OK",
        "fields": null
      }
    }
  ],
  "status": {
    "code": 0,
    "name": "OK",
    "desc": "OK",
    "fields": null
  },
  "fatal": false
}

Task /Applications/Keybase.app/Contents/SharedSupport/bin/keybase -d --log-format=file install --format=json --components=updater (err): 2016-09-04T12:23:33.816933 ▶ [DEBU keybase globals.go:285] 001 Created LinkCache, max size: 65536, clean dur: 1m0s
2016-09-04T12:23:33.819485 ▶ [DEBU keybase ui.go:557] 002 Can't set GPG_TTY; discover failed
2016-09-04T12:23:33.819545 ▶ [DEBU keybase globals.go:266] 003 Keybase CLI 1.0.17-20160829110024+c4e8dc8
2016-09-04T12:23:33.819566 ▶ [DEBU keybase globals.go:266] 004 - Built with go1.7
2016-09-04T12:23:33.819579 ▶ [DEBU keybase globals.go:266] 005 - Visit https://keybase.io for more details
2016-09-04T12:23:33.819598 ▶ [DEBU keybase main.go:144] 006 + configureProcesses
2016-09-04T12:23:33.819624 ▶ [DEBU keybase main.go:242] 007 + configureLogging
2016-09-04T12:23:33.819648 ▶ [DEBU keybase main.go:251] 008 Disabling log forwarding
2016-09-04T12:23:33.819661 ▶ [DEBU keybase main.go:244] 009 - configureLogging
2016-09-04T12:23:33.819673 ▶ [DEBU keybase main.go:146] 00a - configureProcesses -> <nil>
2016-09-04T12:23:33.819834 ▶ [DEBU keybase install_osx.go:401] 00b Installing components: [updater]
2016-09-04T12:23:33.819955 ▶ [DEBU keybase install_osx.go:841] 00c Using updater path: /Applications/Keybase.app/Contents/SharedSupport/bin/updater
2016-09-04T12:23:33.827977 ▶ [DEBU keybase install_osx.go:857] 00d No status, needs install
2016-09-04T12:23:33.828041 ▶ [INFO keybase launchd.go:137] 00e Stopping keybase.updater
2016-09-04T12:23:33.833374 ▶ [DEBU keybase launchd.go:140] 00f Output (launchctl remove): 
2016-09-04T12:23:33.833444 ▶ [DEBU keybase launchd.go:145] 010 Nothing to stop (keybase.updater)
2016-09-04T12:23:33.833545 ▶ [INFO keybase launchd.go:251] 011 Removing /Users/xxxxx/Library/LaunchAgents/keybase.updater.plist
2016-09-04T12:23:33.834255 ▶ [DEBU keybase install_osx.go:874] 012 Installing updater service
2016-09-04T12:23:33.834422 ▶ [INFO keybase launchd.go:234] 013 Saving /Users/xxxxxx/Library/LaunchAgents/keybase.updater.plist
2016-09-04T12:23:33.834532 ▶ [DEBU keybase util.go:173] 014 + Writing to /Users/xxxxx/Library/LaunchAgents/keybase.updater.plist
2016-09-04T12:23:33.835361 ▶ [DEBU keybase util.go:175] 015 | Temporary file generated: /Users/xxxxx/Library/LaunchAgents/keybase.updater.plist.DZKRR463OV2F6UPLHSCY54NLXUHI64RH
2016-09-04T12:23:33.835677 ▶ [DEBU keybase util.go:194] 016 - Wrote to /Users/xxxxx/Library/LaunchAgents/keybase.updater.plist -> ok
2016-09-04T12:23:33.835771 ▶ [INFO keybase launchd.go:93] 017 Starting keybase.updater
2016-09-04T12:23:33.850411 ▶ [DEBU keybase launchd.go:96] 018 Output (launchctl load): 
2016-09-04T12:23:33.866475 ▶ [DEBU keybase launchd.go:109] 019 Service status: &launchd.ServiceStatus{label:"keybase.updater", pid:"6234", lastExitStatus:""}
2016-09-04T12:23:33.879687 ▶ [DEBU keybase out_of_date.go:13] 01a + PrintOutOfDateWarnings
2016-09-04T12:23:33.879735 ▶ [DEBU keybase util.go:386] 01b + GetSocket
2016-09-04T12:23:33.879756 ▶ [DEBU keybase util.go:387] 01c - GetSocket -> ok
2016-09-04T12:23:33.879771 ▶ [DEBU keybase socket.go:69] 01d | empty socket wrapper; need a new one
2016-09-04T12:23:33.879808 ▶ [DEBU keybase socket_nix.go:23] 01e Dialing unix:/Users/xxxxx/Library/Caches/Keybase/keybased.sock
2016-09-04T12:23:33.880116 ▶ [DEBU keybase socket.go:84] 01f | DialSocket -> ERROR: dial unix /Users/xxxxx/Library/Caches/Keybase/keybased.sock: connect: no such file or directory
2016-09-04T12:23:33.880149 ▶ [DEBU keybase out_of_date.go:18] 020 Ignoring error in printOutOfDateWarnings: dial unix /Users/xxxxx/Library/Caches/Keybase/keybased.sock: connect: no such file or directory
2016-09-04T12:23:33.880170 ▶ [DEBU keybase out_of_date.go:19] 021 - PrintOutOfDateWarnings
2016-09-04T12:23:33.981339 ▶ [DEBU keybase globals.go:313] 022 Calling shutdown first time through
2016-09-04T12:23:33.981440 ▶ [DEBU keybase login_state.go:1015] 023 + Account "LoginState - Shutdown"
2016-09-04T12:23:33.981502 ▶ [DEBU keybase login_state.go:928] 024 - LoginState: Leaving request loop
2016-09-04T12:23:33.981522 ▶ [DEBU keybase login_state.go:1017] 025 - Account "LoginState - Shutdown"
2016-09-04T12:23:33.981556 ▶ [DEBU keybase globals.go:361] 026 exiting shutdown code=0; err=<nil>

Task dispatch completion
Checking launchd status for updater
Task: /Applications/Keybase.app/Contents/SharedSupport/bin/keybase --log-format=file launchd status --format=json updater
Task termination handler, reading output
Task /Applications/Keybase.app/Contents/SharedSupport/bin/keybase --log-format=file launchd status --format=json updater (out): {
  "version": "",
  "label": "keybase.updater",
  "pid": "6234",
  "lastExitStatus": "",
  "bundleVersion": "",
  "installStatus": 4,
  "installAction": 1,
  "status": {
    "code": 0,
    "name": "OK",
    "desc": "OK",
    "fields": null
  }
}

Task dispatch completion
Install: Service
Task: /Applications/Keybase.app/Contents/SharedSupport/bin/keybase -d --log-format=file install --format=json --components=service
Task /Applications/Keybase.app/Contents/SharedSupport/bin/keybase -d --log-format=file install --format=json --components=service (out): {
  "componentResults": [
    {
      "name": "service",
      "status": {
        "code": 0,
        "name": "OK",
        "desc": "OK",
        "fields": null
      }
    }
  ],
  "status": {
    "code": 0,
    "name": "OK",
    "desc": "OK",
    "fields": null
  },
  "fatal": false
}

Task /Applications/Keybase.app/Contents/SharedSupport/bin/keybase -d --log-format=file install --format=json --components=service (err): 2016-09-04T12:23:34.139010 ▶ [DEBU keybase globals.go:285] 001 Created LinkCache, max size: 65536, clean dur: 1m0s
2016-09-04T12:23:34.142109 ▶ [DEBU keybase ui.go:557] 002 Can't set GPG_TTY; discover failed
2016-09-04T12:23:34.142178 ▶ [DEBU keybase globals.go:266] 003 Keybase CLI 1.0.17-20160829110024+c4e8dc8
2016-09-04T12:23:34.142199 ▶ [DEBU keybase globals.go:266] 004 - Built with go1.7
2016-09-04T12:23:34.142212 ▶ [DEBU keybase globals.go:266] 005 - Visit https://keybase.io for more details
2016-09-04T12:23:34.142231 ▶ [DEBU keybase main.go:144] 006 + configureProcesses
2016-09-04T12:23:34.142254 ▶ [DEBU keybase main.go:242] 007 + configureLogging
2016-09-04T12:23:34.142270 ▶ [DEBU keybase main.go:251] 008 Disabling log forwarding
2016-09-04T12:23:34.142291 ▶ [DEBU keybase main.go:244] 009 - configureLogging
2016-09-04T12:23:34.142305 ▶ [DEBU keybase main.go:146] 00a - configureProcesses -> <nil>
2016-09-04T12:23:34.142479 ▶ [DEBU keybase install_osx.go:401] 00b Installing components: [service]
2016-09-04T12:23:34.142575 ▶ [DEBU keybase install_osx.go:479] 00c Using binPath: /Applications/Keybase.app/Contents/SharedSupport/bin/keybase
2016-09-04T12:23:34.142662 ▶ [DEBU keybase install_osx.go:487] 00d Checking service: keybase.service
2016-09-04T12:23:35.164809 ▶ [DEBU keybase install_osx.go:489] 00e Service: Not Installed (Action: Install); keybase1.ServiceStatus{Version:"", Label:"keybase.service", Pid:"", LastExitStatus:"", BundleVersion:"1.0.17-20160829110024+c4e8dc8", InstallStatus:2, InstallAction:4, Status:keybase1.Status{Code:0, Name:"OK", Desc:"", Fields:[]keybase1.StringKVPair(nil)}}
2016-09-04T12:23:35.165129 ▶ [INFO keybase launchd.go:137] 00f Stopping keybase.service
2016-09-04T12:23:35.174026 ▶ [DEBU keybase launchd.go:140] 010 Output (launchctl remove): 
2016-09-04T12:23:35.174245 ▶ [DEBU keybase launchd.go:145] 011 Nothing to stop (keybase.service)
2016-09-04T12:23:35.174434 ▶ [INFO keybase launchd.go:251] 012 Removing /Users/xxxxx/Library/LaunchAgents/keybase.service.plist
2016-09-04T12:23:35.174621 ▶ [INFO keybase launchd.go:137] 013 Stopping homebrew.mxcl.keybase
2016-09-04T12:23:35.180848 ▶ [DEBU keybase launchd.go:140] 014 Output (launchctl remove): 
2016-09-04T12:23:35.181037 ▶ [DEBU keybase launchd.go:145] 015 Nothing to stop (homebrew.mxcl.keybase)
2016-09-04T12:23:35.181316 ▶ [DEBU keybase install_osx.go:505] 016 Installing Keybase service
2016-09-04T12:23:35.181636 ▶ [INFO keybase launchd.go:234] 017 Saving /Users/xxxxx/Library/LaunchAgents/keybase.service.plist
2016-09-04T12:23:35.181713 ▶ [DEBU keybase util.go:173] 018 + Writing to /Users/xxxxx/Library/LaunchAgents/keybase.service.plist
2016-09-04T12:23:35.182559 ▶ [DEBU keybase util.go:175] 019 | Temporary file generated: /Users/xxxxx/Library/LaunchAgents/keybase.service.plist.66WLZW7OTLN445ABURHZ2OGJAEWAKCHR
2016-09-04T12:23:35.182808 ▶ [DEBU keybase util.go:194] 01a - Wrote to /Users/xxxxx/Library/LaunchAgents/keybase.service.plist -> ok
2016-09-04T12:23:35.182897 ▶ [INFO keybase launchd.go:93] 01b Starting keybase.service
2016-09-04T12:23:35.192160 ▶ [DEBU keybase launchd.go:96] 01c Output (launchctl load): 
2016-09-04T12:23:35.206015 ▶ [DEBU keybase launchd.go:109] 01d Service status: &launchd.ServiceStatus{label:"keybase.service", pid:"6246", lastExitStatus:""}
2016-09-04T12:23:35.223614 ▶ [DEBU keybase service_info.go:92] 01e Waiting for service info file (/Users/xxxxx/Library/Caches/Keybase/keybased.info)...
2016-09-04T12:23:35.725881 ▶ [DEBU keybase service_info.go:106] 01f Found service info: libkb.ServiceInfo{Version:"1.0.17-20160829110024+c4e8dc8", Label:"keybase.service", Pid:6246}
2016-09-04T12:23:35.726190 ▶ [DEBU keybase out_of_date.go:13] 020 + PrintOutOfDateWarnings
2016-09-04T12:23:35.726230 ▶ [DEBU keybase util.go:386] 021 + GetSocket
2016-09-04T12:23:35.726258 ▶ [DEBU keybase util.go:387] 022 - GetSocket -> ok
2016-09-04T12:23:35.726276 ▶ [DEBU keybase socket.go:69] 023 | empty socket wrapper; need a new one
2016-09-04T12:23:35.726309 ▶ [DEBU keybase socket_nix.go:23] 024 Dialing unix:/Users/xxxxx/Library/Caches/Keybase/keybased.sock
2016-09-04T12:23:35.726475 ▶ [DEBU keybase socket.go:84] 025 | DialSocket -> ok
SIGABRT: abort
PC=0x405dee3 m=0

goroutine 0 [idle]:
runtime.kevent(0x7fff00000005, 0x0, 0x7fff00000000, 0x7fff5fbff1b8, 0x40, 0x0, 0x0, 0x7bcdb2c0, 0x0, 0x0, ...)
	/usr/local/Cellar/go/1.7/libexec/src/runtime/sys_darwin_amd64.s:499 +0x23
runtime.netpoll(0x5176501, 0x0)
	/usr/local/Cellar/go/1.7/libexec/src/runtime/netpoll_kqueue.go:75 +0xb6
runtime.findrunnable(0xc420019500, 0x0)
	/usr/local/Cellar/go/1.7/libexec/src/runtime/proc.go:2003 +0x28c
runtime.schedule()
	/usr/local/Cellar/go/1.7/libexec/src/runtime/proc.go:2120 +0x14c
runtime.park_m(0xc4202ad6c0)
	/usr/local/Cellar/go/1.7/libexec/src/runtime/proc.go:2183 +0x123
runtime.mcall(0x7fff5fbffb30)
	/usr/local/Cellar/go/1.7/libexec/src/runtime/asm_amd64.s:240 +0x5b

goroutine 1 [select]:
github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc.(*dispatch).Call(0xc42032f480, 0x5131b00, 0xc42000e298, 0x4ba8c0b, 0x19, 0x49d2c80, 0xc420361900, 0x0, 0x0, 0x512c9c0, ...)
	/Users/test/go/src/github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc/dispatch.go:70 +0x915
github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc.(*Client).Call(0xc4203618c0, 0x5131b00, 0xc42000e298, 0x4ba8c0b, 0x19, 0x49d2c80, 0xc420361900, 0x0, 0x0, 0xc4202437e8, ...)
	/Users/test/go/src/github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc/client.go:37 +0x10d
github.com/keybase/client/go/protocol/keybase1.ConfigClient.HelloIAm(0x512b7c0, 0xc4203618c0, 0x5131b00, 0xc42000e298, 0x185f, 0x1, 0xc42000c120, 0x6, 0x6, 0x0, ...)
	/Users/test/go/src/github.com/keybase/client/go/protocol/keybase1/config.go:374 +0x1d9
github.com/keybase/client/go/client.introduceMyself(0xc4201b46c0, 0x5136d80, 0xc420332d80, 0xc420134328, 0x5136d80)
	/Users/test/go/src/github.com/keybase/client/go/client/rpc.go:274 +0x17e
github.com/keybase/client/go/client.getSocket(0xc4201b46c0, 0x0, 0xc420332900, 0x0, 0x0, 0x2b48c8c7040140e8)
	/Users/test/go/src/github.com/keybase/client/go/client/rpc.go:29 +0xb5
github.com/keybase/client/go/client.getSocketNoRetry(0xc4201b46c0, 0x51761a0, 0xc420243a88, 0x47c9e80, 0xc4201d41e0)
	/Users/test/go/src/github.com/keybase/client/go/client/rpc.go:22 +0x30
github.com/keybase/client/go/client.GetRPCClientWithContext(0xc4201b46c0, 0xc420243ae0, 0xc420243ae8, 0x40e70c6, 0xc420243ae0, 0xc420243af0)
	/Users/test/go/src/github.com/keybase/client/go/client/rpc.go:35 +0x2f
github.com/keybase/client/go/client.GetConfigClient(0xc4201b46c0, 0x513acd0, 0xc4201305c0, 0x4ba696b, 0x18)
	/Users/test/go/src/github.com/keybase/client/go/client/rpc.go:61 +0x2b
github.com/keybase/client/go/client.PrintOutOfDateWarnings(0xc4201b46c0)
	/Users/test/go/src/github.com/keybase/client/go/client/out_of_date.go:16 +0xef
main.mainInner(0xc4201b46c0, 0x4c29488, 0x41215d9)
	/Users/test/go/src/github.com/keybase/client/go/keybase/main.go:135 +0x623
main.main()
	/Users/test/go/src/github.com/keybase/client/go/keybase/main.go:56 +0x8b

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/Cellar/go/1.7/libexec/src/runtime/asm_amd64.s:2086 +0x1

goroutine 5 [syscall]:
os/signal.signal_recv(0x0)
	/usr/local/Cellar/go/1.7/libexec/src/runtime/sigqueue.go:116 +0x157
os/signal.loop()
	/usr/local/Cellar/go/1.7/libexec/src/os/signal/signal_unix.go:22 +0x22
created by os/signal.init.1
	/usr/local/Cellar/go/1.7/libexec/src/os/signal/signal_unix.go:28 +0x41

goroutine 18 [select]:
github.com/keybase/client/go/libkb.(*LoginState).requests(0xc420254000)
	/Users/test/go/src/github.com/keybase/client/go/libkb/login_state.go:897 +0x6b5
created by github.com/keybase/client/go/libkb.NewLoginState
	/Users/test/go/src/github.com/keybase/client/go/libkb/login_state.go:115 +0x242

goroutine 19 [chan receive]:
main.HandleSignals()
	/Users/test/go/src/github.com/keybase/client/go/keybase/main.go:293 +0x175
created by main.main
	/Users/test/go/src/github.com/keybase/client/go/keybase/main.go:55 +0x7d

goroutine 7 [select, locked to thread]:
runtime.gopark(0x4c29c28, 0x0, 0x4b8f0fe, 0x6, 0x18, 0x2)
	/usr/local/Cellar/go/1.7/libexec/src/runtime/proc.go:259 +0x13a
runtime.selectgoImpl(0xc420026f30, 0x0, 0x18)
	/usr/local/Cellar/go/1.7/libexec/src/runtime/select.go:423 +0x11d9
runtime.selectgo(0xc420026f30)
	/usr/local/Cellar/go/1.7/libexec/src/runtime/select.go:238 +0x1c
runtime.ensureSigM.func1()
	/usr/local/Cellar/go/1.7/libexec/src/runtime/signal1_unix.go:304 +0x2d1
runtime.goexit()
	/usr/local/Cellar/go/1.7/libexec/src/runtime/asm_amd64.s:2086 +0x1

goroutine 20 [select]:
github.com/keybase/client/go/vendor/stathat.com/c/ramcache.(*Ramcache).cleanup(0xc42014ce40)
	/Users/test/go/src/github.com/keybase/client/go/vendor/stathat.com/c/ramcache/ramcache.go:164 +0x13f
created by github.com/keybase/client/go/vendor/stathat.com/c/ramcache.New
	/Users/test/go/src/github.com/keybase/client/go/vendor/stathat.com/c/ramcache/ramcache.go:44 +0x179

goroutine 21 [select]:
github.com/keybase/client/go/vendor/stathat.com/c/ramcache.(*Ramcache).cleanup(0xc42014cf00)
	/Users/test/go/src/github.com/keybase/client/go/vendor/stathat.com/c/ramcache/ramcache.go:164 +0x13f
created by github.com/keybase/client/go/vendor/stathat.com/c/ramcache.New
	/Users/test/go/src/github.com/keybase/client/go/vendor/stathat.com/c/ramcache/ramcache.go:44 +0x179

goroutine 22 [select]:
github.com/keybase/client/go/vendor/stathat.com/c/ramcache.(*Ramcache).cleanup(0xc42014cfc0)
	/Users/test/go/src/github.com/keybase/client/go/vendor/stathat.com/c/ramcache/ramcache.go:164 +0x13f
created by github.com/keybase/client/go/vendor/stathat.com/c/ramcache.New
	/Users/test/go/src/github.com/keybase/client/go/vendor/stathat.com/c/ramcache/ramcache.go:44 +0x179

goroutine 23 [select]:
github.com/keybase/client/go/libkb.(*LinkCache).handle(0xc42014d2c0)
	/Users/test/go/src/github.com/keybase/client/go/libkb/link_cache.go:136 +0x110d
created by github.com/keybase/client/go/libkb.NewLinkCache
	/Users/test/go/src/github.com/keybase/client/go/libkb/link_cache.go:49 +0x2d4

goroutine 24 [select]:
github.com/keybase/client/go/libkb.(*LinkCache).periodic(0xc42014d2c0)
	/Users/test/go/src/github.com/keybase/client/go/libkb/link_cache.go:204 +0x123
created by github.com/keybase/client/go/libkb.NewLinkCache
	/Users/test/go/src/github.com/keybase/client/go/libkb/link_cache.go:50 +0x2f6

goroutine 67 [select]:
github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc.(*framedMsgpackEncoder).writerLoop(0xc42032f440)
	/Users/test/go/src/github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc/codec.go:141 +0x1a4
created by github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc.newFramedMsgpackEncoder
	/Users/test/go/src/github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc/codec.go:76 +0x18e

goroutine 68 [select]:
github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc.(*receiveHandler).taskLoop(0xc42036a840)
	/Users/test/go/src/github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc/receiver.go:55 +0x32b
created by github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc.newReceiveHandler
	/Users/test/go/src/github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc/receiver.go:48 +0x23d

goroutine 69 [IO wait]:
net.runtime_pollWait(0x52b5728, 0x72, 0x4)
	/usr/local/Cellar/go/1.7/libexec/src/runtime/netpoll.go:160 +0x59
net.(*pollDesc).wait(0xc42034aae0, 0x72, 0xc420039b18, 0xc42000e058)
	/usr/local/Cellar/go/1.7/libexec/src/net/fd_poll_runtime.go:73 +0x38
net.(*pollDesc).waitRead(0xc42034aae0, 0x512a300, 0xc42000e058)
	/usr/local/Cellar/go/1.7/libexec/src/net/fd_poll_runtime.go:78 +0x34
net.(*netFD).Read(0xc42034aa80, 0xc42035d000, 0x1000, 0x1000, 0x0, 0x512a300, 0xc42000e058)
	/usr/local/Cellar/go/1.7/libexec/src/net/fd_unix.go:243 +0x1a1
net.(*conn).Read(0xc420134328, 0xc42035d000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/Cellar/go/1.7/libexec/src/net/net.go:173 +0x70
bufio.(*Reader).fill(0xc42036a2a0)
	/usr/local/Cellar/go/1.7/libexec/src/bufio/bufio.go:97 +0x10c
bufio.(*Reader).ReadByte(0xc42036a2a0, 0x0, 0x0, 0x0)
	/usr/local/Cellar/go/1.7/libexec/src/bufio/bufio.go:231 +0x51
github.com/keybase/client/go/vendor/github.com/keybase/go-codec/codec.(*ioDecReader).readn1(0xc42020e970, 0x0)
	/Users/test/go/src/github.com/keybase/client/go/vendor/github.com/keybase/go-codec/codec/decode.go:286 +0x37
github.com/keybase/client/go/vendor/github.com/keybase/go-codec/codec.(*msgpackDecDriver).readNextBd(0xc420344c80)
	/Users/test/go/src/github.com/keybase/client/go/vendor/github.com/keybase/go-codec/codec/msgpack.go:560 +0x34
github.com/keybase/client/go/vendor/github.com/keybase/go-codec/codec.(*msgpackDecDriver).TryDecodeAsNil(0xc420344c80, 0x0)
	/Users/test/go/src/github.com/keybase/client/go/vendor/github.com/keybase/go-codec/codec/msgpack.go:587 +0x51
github.com/keybase/client/go/vendor/github.com/keybase/go-codec/codec.(*Decoder).decode(0xc42020e900, 0x49ad860, 0xc42000f640)
	/Users/test/go/src/github.com/keybase/client/go/vendor/github.com/keybase/go-codec/codec/decode.go:1512 +0x66
github.com/keybase/client/go/vendor/github.com/keybase/go-codec/codec.(*Decoder).Decode(0xc42020e900, 0x49ad860, 0xc42000f640, 0x0, 0x0)
	/Users/test/go/src/github.com/keybase/client/go/vendor/github.com/keybase/go-codec/codec/decode.go:1421 +0x84
github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc.(*packetHandler).loadNextFrame(0xc42032f4c0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/Users/test/go/src/github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc/packetizer.go:61 +0x7a
github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc.(*packetHandler).NextFrame(0xc42032f4c0, 0x0, 0x1, 0x0, 0x0)
	/Users/test/go/src/github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc/packetizer.go:34 +0x40
github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc.(*transport).receiveFramesLoop(0xc420332d80)
	/Users/test/go/src/github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc/transport.go:145 +0x62
github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc.(*transport).receiveFrames.func1(0xc420332d80)
	/Users/test/go/src/github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc/transport.go:117 +0x2b
created by github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc.(*transport).receiveFrames
	/Users/test/go/src/github.com/keybase/client/go/vendor/github.com/keybase/go-framed-msgpack-rpc/transport.go:118 +0x87

rax    0x4
rbx    0x5176528
rcx    0x7fff5fbff148
rdx    0x0
rdi    0x5
rsi    0x0
rbp    0x7fff5fbff9b8
rsp    0x7fff5fbff148
r8     0x40
r9     0x0
r10    0x7fff5fbff1b8
r11    0x246
r12    0x0
r13    0xf6
r14    0xa
r15    0x2
rip    0x405dee3
rflags 0x247
cs     0x7
fs     0x0
gs     0x0

Task timed out: /Applications/Keybase.app/Contents/SharedSupport/bin/keybase -d --log-format=file install --format=json --components=service
Task termination handler, timed out, skipping
Install complete
Privileged Helper: Installed, Bundle Version: 1.0.29
Version: 1.0.29
Updater: Installed, PID: 6234
Service: Install Error: Task timed out: /Applications/Keybase.app/Contents/SharedSupport/bin/keybase -d --log-format=file install --format=json --components=service
```

**Comment by cjb** [comment]
Thanks for the report, CC @gabriel

**Comment by echtniels** [comment]
Unless it was you that pushed a bug-fix, it appears installing yesterday's OSX patches fixed this issue. Now have full control over my folders and even the taskbar gui works!

**Comment by gabriel** [comment]
This might have a been a transient error. Re-opening so I can take a look.

**Comment by gabriel** [comment]
This is fixed in latest version released last week (1.0.17-20160913233827+18b5a25).
